src/api/spec/components/bs_request_overview_avatars_component_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe BsRequestOverviewAvatarsComponent, type: :component do
+  describe '#package_avatar_objects' do
+    let(:user) { create(:user, login: 'King') }
+    let(:review) { create(:review, by_project: package.project.name, by_package: package.name) }
+    let(:project) { create(:project_with_package, name: 'Apache', package_name: 'apache2') }
+    let(:package) { project.packages.first }
+    let(:other_user) { create(:user, login: 'bob', realname: 'Bob') }
+
+    context 'when we have a review by_package' do
+      context 'and we have a maintainer assigned to the package' do
+        let(:maintainer) { Role.hashed['maintainer'] }
+
+        before do
+          package.relationships.create(user: other_user, role: maintainer)
```

Comment:
Nitpick: we could use the [relationship factory](https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/factories/relationship.rb)...

Reply:
Hm, yeah